src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -1176,20 +1178,20 @@ private static void doCopyFile(final File srcFile, final File destFile, final bo
      * @param srcDir  an existing directory to copy, must not be {@code null}
      * @param destDir the directory to place the copy in, must not be {@code null}
      *
-     * @throws NullPointerException if source or destination is {@code null}
+     * @throws IllegalArgumentException if source or destination is {@code null}
      * @throws IOException          if source or destination is invalid
      * @throws IOException          if an IO error occurs during copying
      * @since 1.2
      */
     public static void copyDirectoryToDirectory(final File srcDir, final File destDir) throws IOException {
         if (srcDir == null) {
-            throw new NullPointerException("Source must not be null");
+            throw new IllegalArgumentException("Source must not be null");
```

Comment:
Do not change these please. Since Java 8 and the broad use of [java.util.Objects.requireNonNull(T, String)](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#requireNonNull-T-java.lang.String-), any edits to current throws of NPE should use the aforementioned API.